test/http-test.js

```diff
@@ -338,6 +341,49 @@ describe('HTTP', function() {
     }
   });
 
+  it('should generate 10 blocks from RPC call', async () => {
+    blocks = await nclient.execute(
+      'generatetoaddress',
+      [10, addr.toString('regtest')]
+    );
+    assert.strictEqual(blocks.length, 10);
+  });
+
+  it('should initiate rescan from socket without a bloom filter', async () => {
+    // Rescan from height 5. Without a filter loaded = no response, but no error
+    const response = await nclient.call('rescan', 5);
+    assert.strictEqual(null, response);
+  });
+
+  it('should initiate rescan from socket WITH a bloom filter', async () => {
+    // Create an SPV-standard Bloom filter and add one of our wallet addresses
+    const filter = BloomFilter.fromRate(20000, 0.001, BloomFilter.flags.ALL);
+    filter.add(addr.toString('regtest'), 'ascii');
+
+    // Send Bloom filter to server
+    await nclient.call('set filter', filter.filter);
+
+    // Rescan is a weird call, the server sends a socket event BACK to the
```

Comment:
Can you update this message to use more imperative language?

Reply:
Comment has been updated